app/celery/process_ga4_measurement_tasks.py

```diff
@@ -72,6 +55,18 @@ def post_to_ga4(
         current_app.logger.error('GA4_MEASUREMENT_ID is not set')
         return False
 
+    # Retrieve the notification from the database
+    stmt = select(Notification).where(Notification.id == notification_id)
```

Comment:
Instead of doing `select(Notification).where (Notification.id == notification_id)` then `.first()`, could we do a `notification = db.session.get(Notification, notification_id)` since we are querying for a specific primary key? 

I think it's faster and if there's no Notification Object - it should return None.

Reply:
hell yeah, thanks for that.